conda/plan.py

```diff
@@ -9,22 +9,28 @@
 """
```

Comment:
This module has a lot of new code.  We definitely need to spend some time on writing tests here.